### DISCUSSION TITLE
Not really new at Github but the community is  something new I discovered on Github

### DISCUSSION THREAD

**Discussion by moozunch** [community/community#111100] [New to GitHub]
Hello everyone, nice to meet y'all! I'm the definetely not new at Github but i recently discover a lot of new things like community after I got my Github student acceptance. I hope I could make a good use of this opportunity and meet a lot of y'all great community, I hope I can learn a lot from you guys ^____^!!

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @moozunch  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be closed to keep our community nice and tidy 🧹.
